mne_bids/write.py

```diff
@@ -309,8 +319,12 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False, verbose=True):
     meas_date = raw.info['meas_date']
     if isinstance(meas_date, (tuple, list, np.ndarray)):
         meas_date = meas_date[0]
-        acq_time = datetime.fromtimestamp(
-            meas_date).strftime('%Y-%m-%dT%H:%M:%S')
+        # windows datetime bug for timestamp < 0
+        # OSError [Errno 22] Invalid Argument
+        acq_time = \
+            (datetime.fromtimestamp(0) +
+                timedelta(seconds=int(meas_date))).strftime(
+                    '%Y-%m-%dT%H:%M:%S')
```

Comment:
same comment here, I'm not sure what's going on here

Reply:
It's the same as before, just written with fromtimestamp(0) and then shifting rather than the actual timestamp because of windows errors (see "For the record..." above).